DuckDuckGo/AppDependencyProvider.swift

```diff
@@ -134,13 +134,11 @@ class AppDependencyProvider: DependencyProvider {
             purchasePlatform: .appStore)
         let accessTokenProvider: () -> String? = {
             func isSubscriptionEnabled() -> Bool {
-                if let subscriptionOverrideEnabled = UserDefaults.networkProtectionGroupDefaults.subscriptionOverrideEnabled {
 #if ALPHA || DEBUG
+                if let subscriptionOverrideEnabled = UserDefaults.networkProtectionGroupDefaults.subscriptionOverrideEnabled {
                     return subscriptionOverrideEnabled
-#else
-                    return false
```

Comment:
Release users shouldn't ever go in here, so let's just clean things up so they won't be allowed to.